source/reference/mongosync.txt

```diff
@@ -27,7 +27,14 @@ line options.
 .. note::
 
    - Both the source cluster and destination cluster must use MongoDB
-     6.0 or later.
+     6.0 or later. 
+     
+     You can migrate data on clusters (source) with versions of MongoDB
+     lower than 6.0 one time to an Atlas cluster. But this requires
```

Comment:
I'd re-write the sentence to avoid starting with "But...", perhaps something like "Migration from clusters with a lower version requires additional preperation..." or something along those lines.